app/javascript/components/constants.js

```diff
@@ -1 +1,2 @@
 export const guitarsUrl = '/guitars';
+export const guitarUrl = '/guitars/:id';
```

Comment:
usually, constants are written in capital letters.

```suggestion
export const GUITARS_URL = '/guitars';
export const GUITAR_URL = '/guitars/:id';
```

Reply:
Fixed 👍🏼